### ISSUE TITLE
Trying to kill long-running (blocked) query causes beachball and non-responsive process

### ISSUE THREAD

**Issue by ghost** [sequelpro/sequelpro#1674]
Issue imported from Google Code: https://code.google.com/p/sequel-pro/issues/detail?id=1674
Reporter:john.bur...@alumni.brown.edu Date:2013-03-21 14:19:25 Status:WontFix Closed:2013-05-19

<b>What steps will reproduce the problem?</b>
1.  connect to server
2.  start update query that is blocked by a table locked to another LONG running query
3.  after a long while, Click the STOP button in the dialog box
This starts the infamous spinning beachball.
4.  after another long while, go to the server and kill the mysql process.
Now the socket is closed on the server, but still open on the client.
And there is still the beachball.  All other Sequel Pro windows and tabs are also frozen.
5.. had to Force Quit.

What is the expected output?  In 3, I would expect it to cancel the query, and return control to me.
What do you see instead? Spinning beach ball;  

What version of Sequel Pro are you using? 1.0.1
What version of OS X are you using?  10.8.2
What version of MySQL are you connecting to on the server? 5.5

<b>Please provide any additional information below.</b>
